src/components/atoms/DatePicker/DatePicker.tsx

```diff
@@ -4,14 +4,28 @@ import DatePicker from "react-datepicker";
 import "react-datepicker/dist/react-datepicker.css";
 import { DatePickerDay } from "./DatePickerDay/DatePickerDay";
 
-interface CustomDatePickerProps {
+export interface CustomDatePickerProps {
   isEndDate?: boolean;
   additionalDatePickerClasses?: string;
+  calendarStartFromMondayDay?: boolean;
+  startDatePlaceHolder?: string;
+  endDatePlaceHolder?: string;
+  startDateHeading?:string;
+  endDateHeading?:string;
+  endDateLimit?: Date;
+  markSundayAsRed ?: boolean;
 }
 
 export function CustomDatePicker({
   isEndDate,
   additionalDatePickerClasses = "",
+  calendarStartFromMondayDay,
+  startDatePlaceHolder,
+  startDateHeading,
+  endDateHeading,
+  endDatePlaceHolder,
+  endDateLimit,
+  markSundayAsRed
 }: CustomDatePickerProps) {
   const [startDate, setStartDate] = useState<Date>(new Date());
   const [isStartDateOpen, setIsStartDateOpen] = useState<boolean>(false);
```

Comment:
`isStartDateOpen` this variable is not in use, we can remove if this state is not needed.

Reply:
Same for `isEndDateOpen`

Reply:
updated;